### PR TITLE
Create Bounty Snake README

### PR DESCRIPTION
**Pull request by aforsilly**
I did a thing. Hopefully I did it correctly/put it in the right place :)

### CODE REVIEW

File: 2019-Bounty-Snakes/README.me [hunk 1]
```diff
@@ -0,0 +1,42 @@
+#2019 Bounty Snakes
```
Comment (joram) [inline 2019-Bounty-Snakes/README.me]: this file extension is probably meant to be `md`?

File: 2019-Bounty-Snakes/README.me [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+Defeat a 2019 Bounty Snake on March 2 for a chance to win a prize! Prizes and snake descriptions listed below. 
+
+### [Dyspatch by Sendwithus](https://www.dyspatch.io/sendwithus/)
```
Comment (joram) [inline 2019-Bounty-Snakes/README.me]: why jump from `#` to `###`? skipping `##` bolding?
also, is there a particular order to these snakes?

Comment (aforsilly) [reply 2019-Bounty-Snakes/README.me]: No particular order. Open to changing the styling!

Comment (joram) [reply 2019-Bounty-Snakes/README.me]: well, that can be another pull request later, or done in this one now, which ever you want. I think it's an improvement to what we had (which was nothing)

**Comment by aforsilly** [comment]
Ok fixed the styling & file name.

**Comment by codeallthethingz** [comment]
Merged!

**Pull request merged**